index.html

```diff
@@ -31,6 +31,7 @@ <h2> Choose Grid Size </h2>
       </form>
       <h2> Design Canvas </h2>
       <table id="pixel_canvas"> </table>
+      <div id="b-margin"></div>
       <script src="js/app.js"></script>
 
    
```

Comment:
same with this, add a single line at the end. Thanks for your contribution! @Alik-Kumar-Ghosh

Reply:
Please check made the changes needed.

Reply:
I meant at the **end of the file** i.e. after </html> -- an empty line, you can undo the changes that you did.

Reply:
ok